src/epd7in5_v2/mod.rs

```diff
@@ -59,7 +59,7 @@ where
         delay: &mut DELAY,
     ) -> Result<(), SPI::Error> {
         // Reset the device
-        self.interface.reset(delay);
+        self.interface.reset(delay, 4);
```

Comment:
https://github.com/waveshare/e-Paper/blob/751a9fb93fdd486511222777b0070c51bf436386/RaspberryPi%26JetsonNano/python/lib/waveshare_epd/epd7in5_V2.py#L52

I think it should be only 2 ms for this display. You might have looked for the B/C variants (red/yellow) which have 4 ms.